### ISSUE TITLE
Edit emails before sending a message

### ISSUE THREAD

**Issue by SimonLab** [FAC-GM/app#273]
As a user I want to be able to add or delete some email on the email page.

**Comment by heron2014** [comment]
The basic solution for this feature is : 

![email-editable](https://cloud.githubusercontent.com/assets/8476884/13180572/f08a3822-d720-11e5-80b2-c9c687274fd0.png)

Current populated boxes with email as below would be replaced by above screenshot.

![emails-boxes](https://cloud.githubusercontent.com/assets/8476884/13180635/2b21cb62-d721-11e5-9395-d146011ddef1.png)

**Comment by iteles** [comment]
Agreed, this is exactly what we had discussed 👍🏻 Nicely done on going the extra mile and putting in some visuals @heron2014! 😄
